import-scripts/generate_cdd_attribute_tsv.py

```diff
@@ -81,7 +83,9 @@
 
 # Extracts credentials for later use
 def get_github_credentials(credentials_file):
-    credentials_line = open(credentials_file,"rb").read().rstrip()
+    print("start func")
```

Comment:
looks like some debugging / status lines were left in ... these should be removed